core/src/main/java/com/novoda/noplayer/Heart.java

```diff
@@ -30,6 +30,10 @@ public void startBeatingHeart() {
         handler.post(heartbeat);
     }
 
+    public void bind(Heartbeat onHeartbeat) {
```

Comment:
my bad I meant the null check in the `startBeatingHeart`, or do you think we should be able to beat without providing an action?